lib/mix/tasks/dicon.deploy.ex

```diff
@@ -60,4 +62,16 @@ defmodule Mix.Tasks.Dicon.Deploy do
     Executor.exec(conn, command)
     Executor.exec(conn, ["rm ", release_file])
   end
+
+  defp write_custom_config(conn, name, target_dir, version) do
+    if config = host_config(name) do
+      content = :io_lib.format('~p.\n', [config])
+      target_sub_dir = ["/releases/", version, ?/]
+      config_sub_path = [target_sub_dir, "custom.config"]
+      Executor.write_file(conn, [target_dir, config_sub_path], content)
+      vm_args_path = [target_dir, target_sub_dir, "vm.args"]
+      content = ["-config ", [".", config_sub_path]]
```

Comment:
As far as I can tell, we don't end the `vm.args` file with a newline here. Is this indented behaviour?